File: Assignment_43/program3.c
//////////////////////////////////////////////////////////////////////////
//
// Name :          program3.c
// Discription :   use to Display number of small char form string
// Input :         String 
// Output :        Integer
// Author :        Shivam Rajendra Kale
// Date :          22/07/2025
//
//////////////////////////////////////////////////////////////////////////

/* 
  Start
      Accept input from user
          Display number of small char form string
  Stop    
*/

#include<stdio.h>

 int Small(char *str)
 {
    static int count = 0;

    if(*str != '\0')
    {
        if(*str >= 'a' && *str <= 'z')
        {
            count++;
        }
        
        str++;

        Small(str);
    }
    else 
    {
        return count;
    }  
 }

int main()
{
    int iRet = 0;
    char arr[20] = {'\0'};

    printf("Enter String :");
    scanf("%s",arr);

    iRet = Small(arr);

    printf("Number of characters are :%d\n",iRet);

    return 0;
}

File: Assignment_54/program3.java
//////////////////////////////////////////////////////////////////////////
//
// Name :          program3.java
// Discription :   use to Count capital characters from Array 
// Input :         Integer
// Output :        Integer
// Author :        Shivam Rajendra Kale
// Date :          18/10/2025
//
//////////////////////////////////////////////////////////////////////////

import java.util.*;

class MyArray
{
    public void RevarseArray(char Arr[])
    {
        int i = 0;
        int iCapCount = 0, iSmallCount = 0;
        
        for(i = 0; i < Arr.length; i++)
        {
           if(Arr[i] >= 'A' && Arr[i] <= 'Z')
           {
                iCapCount++;
           }
        }

        System.out.println("Capital characters are : "+iCapCount);
        
    }
}

class program3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int i = 0;

        System.out.println("Enter Array of Characters : ");
        String str = sobj.nextLine();

        char Arr[] = str.toCharArray();

        MyArray mobj = new MyArray();

        mobj.RevarseArray(Arr);

        sobj.close();
    }
}